TracyOpenCL.hpp

```diff
@@ -35,12 +35,28 @@ using TracyCLCtx = void*;
 
 #include <atomic>
 #include <cassert>
+#include <sstream>
 
 #include "Tracy.hpp"
 #include "client/TracyCallstack.hpp"
 #include "client/TracyProfiler.hpp"
 #include "common/TracyAlloc.hpp"
 
+#define TRACY_CL_TO_STRING_INDIRECT(T) #T
+#define TRACY_CL_TO_STRING(T) TRACY_CL_TO_STRING_INDIRECT(T)
```

Comment:
why do you need this indirection?

Reply:
Without the indirection, 
```c
__FILE__ ":" TRACY_CL_TO_STRING(__LINE__)
```
will become something like `"foo.c:__LINE__"`, what we actually want is `"foo.c:42"`